src/newmagic.cpp

```diff
@@ -1700,6 +1708,205 @@ void cast_manipulation_spell(struct char_data *ch, int spell, int force, char *a
     }
     spell_drain(reflected ? vict : ch, spell, force, basedamage);
     break;
+  case SPELL_LASER:
+    if (!check_spell_victim(ch, vict, spell, arg))
+      return;
+    check_killer(ch, vict);
+    if (!AWAKE(vict))
+      target -= 2;
+    else {
+      // NOTE: Added sidestep here. Not sure if you should be able to sidestep lightning, but if you can dodge it in the first place...
+      success -= success_test(GET_DEFENSE(vict) + GET_DEFENSE(vict) ? GET_POWER(vict, ADEPT_SIDESTEP) : 0, 4 + damage_modifier(vict, buf, sizeof(buf)));
+    }
+    act("A thin laser beam is emitted from $n and heads directly towards $N!", TRUE, ch, 0, vict, TO_ROOM);
+    if (!IS_NPC(ch) && PLR_FLAGGED(ch, PLR_KILLER) && !IS_NPC(vict))
+      success = -1;
+    else success += success_test(skill, target + 4);
+    if (success > 0 && GET_REFLECT(vict) && (reflected = reflect_spell(ch, vict, spell, force, 0, target + 4, success))) {
+      vict = ch;
+      ch = temp;      
+    }
+    if (success <= 0) {
+      act("$n easily dodges it, and the beam flickers and vanishes.", FALSE, vict, 0, 0, TO_ROOM);
+      send_to_char("You easily dodge it!\r\n", vict);
+    } else {
+      success -= success_test(GET_BOD(vict) + GET_BODY(vict) + GET_POWER(ch, ADEPT_TEMPERATURE_TOLERANCE), force - (GET_IMPACT(vict) / 2));
+      int dam = convert_damage(stage(success, basedamage));
+      if (!AWAKE(vict)) {
+        act("$n's body recoils in pain as the laser beam finds its mark.", TRUE, vict, 0, 0, TO_ROOM);
+        send_to_char("You feel an uncomfortably painful burning sensation where the beam struck.\r\n", vict);
+      } else if (GET_MENTAL(vict) - (dam * 100) <= 0) {
```

Comment:
It looks like this spell is checking against mental, should it be physical?

Reply:
Yep, definitely should be physical.